lemur/common/validators.py

```diff
@@ -42,20 +42,20 @@ def private_key(key):
 
 def sensitive_domain(domain):
     """
-    Determines if domain has been marked as sensitive.
+    Determines if domain has been marked as sensitive or matches restricted patterns.
     :param domain:
     :return:
     """
+    if SensitiveDomainPermission().can():
+        # User has permission, no need to check anything
+        return
+
     restricted_domains = current_app.config.get('LEMUR_RESTRICTED_DOMAINS', [])
```

Comment:
For us, we generally allow all domains (as validated by issuers) and only blacklist them very specifically. I think at some point we should do away with this setting altogether and rely on the `sensitive` attribute on the domain model. The domains UI in particular has been neglected so it might take a bit of work to get it going.